README.md

```diff
@@ -118,7 +152,11 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+ERD is short for Entity Relationship Diagram. We create them for application as a form of "pseudocoding" or planning out how each entity relates to another.
+
+A Person may have multiple pets, therefore the relationships would be a one-to-many;
+
+
```

Comment:
👍 

Also, to determine the different properties we want to include for a given entity.